daemon/api.go

```diff
@@ -1168,3 +1177,76 @@ func assertsFindMany(c *Command, r *http.Request) Response {
 func getEvents(c *Command, r *http.Request) Response {
 	return EventResponse(c.d.hub)
 }
+
+type changeInfo struct {
+	Kind    string      `json:"kind"`
+	Summary string      `json:"summary"`
+	Status  string      `json:"status"`
+	Tasks   []*taskInfo `json:"tasks,omitempty"`
+}
+
+type taskInfo struct {
+	Kind    string   `json:"kind"`
+	Summary string   `json:"summary"`
+	Status  string   `json:"status"`
+	Log     []string `json:"log,omitempty"`
+}
+
+func getChanges(c *Command, r *http.Request) Response {
+	query := r.URL.Query()
+	logTail := 1
+	qLogTail := query.Get("log-tail")
+	if qLogTail != "" {
+		n, err := strconv.Atoi(qLogTail)
+		if err != nil || n < 0 {
+			return BadRequest("invalid log-tail numeric value")
+		}
+		logTail = n
+	}
+	// XXX: default this to "Done"?
+	excludeStatuses := strings.Split(query.Get("exclude-statuses"), ",")
```

Comment:
I suggest having a more generic parameter such as `select`, which we can set to `all/ready/in-progress`. Defaults to `in-progress`. Otherwise this will blow up when we add/remove statuses.